frontend/website/src/AnnouncementBar.re

```diff
@@ -35,8 +35,25 @@ module Icon = {
 let component = ReasonReact.statelessComponent("AnnouncementBar");
 let make = _ => {
   ...component,
-  render: _self =>
+  render: _self => {
+    // HACK: On firefox (and only firefox) the text seems to not be centered in the announcementbar
+    //  This way we can write CSS that only targets firefox
+    Css.(
+      global(
+        "@-moz-document url-prefix()",
+        [
+          unsafe(
+            "#announcementbar--viewdemo, #announcementbar--testnetlive { margin-bottom",
+            "-0.125rem; }",
+          ),
+          unsafe("#announcementbar--anchor { padding-top", "0.5625rem; }"),
+          unsafe("#announcementbar--anchor { padding-bottom", "0.5625rem; }"),
+        ],
+      )
+    );
```

Comment:
Hopefully we never need to do this again (in which case it will definitely warrant a rewrite lol)